### DISCUSSION TITLE
to squeeze an element

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1013] [Q&A]
Hi! 
I have a list which is in a frame. 
iframe:nth-child(1) |> div:nth-child(9) select
 `select[name="UF_CRM_6333353EB3AB2[]"]
`
It has an element that I need to select.
 `iframe:nth-child(1) |> div:nth-child(9) option:nth-child(7)`
value = 80500

I created a "Click element" and "Trigger event / Keydown", Automa "finds" this item but doesn't "click/select" it.
What to do to "squeeze" this element?

**Comment by Kholid060** [comment]
Use the forms block
![image](https://user-images.githubusercontent.com/22908993/206945997-8dda86d7-175c-404c-985e-168330ecb27d.png)

**Reply by Art-Che** [reply]
Works! Thanks a lot!
